script.js

```diff
@@ -0,0 +1,89 @@
+//create 3 by 3 board
+$(document).ready(function(){
+  // $(function change_cell() {
+      $("td").on("click", function () {
+        var cell_id = $(this).attr('id');
+        console.log(cell_id);
+        console.log($(this));
+        var table = $(this).closest('table');
+        var currentValue = $(this).html();
+        alert(currentValue);
+
+// allow to input X or O
+      var changeValue = prompt("Enter X or O). (Current " + currentValue + ")");
+        $(this).html(changeValue);
+        if (changeValue !== null){
+          $table.find('[id="' + cell_id + '"]').text(changeValue);
```

Comment:
Where did you get "cell_id" that you are searching the DOM for?
You got the id from "this".  "This" is is the thing you clicked on, it's the thing you want to change.  No need to go back and find it again.  You have it in your grasp.